media-gfx/tuxpaint/tuxpaint-0.9.24.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs xdg-utils desktop
+
+DESCRIPTION="Drawing program designed for young children"
+HOMEPAGE="http://www.tuxpaint.org/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+RDEPEND="
```

Comment:
I haven't checked these but it's an important point to make (so why not do it now?) - if we're linking against any libraries and they provide a [subslot](https://devmanual.gentoo.org/general-concepts/slotting/), we should subscribe to it with :=.

Reply:
Checked. No subslots.